doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -231,23 +136,68 @@ General Purpose Equivalents
      - short-circuiting logical OR operator (Python native operator);
        scalar arguments only
 
+   * - ::
+        
+        >> 4 == 4
+        ans = 1
+        >> 4 == 5
+        ans = 0
+
+     - ::
+
+        >>> 4 == 4
+        True
+        >>> 4 == 5
+        False
+
+     - The `boolean objects
+       <https://docs.python.org/3/library/stdtypes.html?highlight=boolean#boolean-values>`_
+       in Python are ``True`` and ``False``, as opposed to MATLAB
+       logical types of ``1`` and ``0``. 
+
+   * - ::
+
+         a=4
+         if a==4
+             fprintf('a = 4\n')
+         elseif a==5
+             fprintf('a = 5\n')
+         end
+
+     - ::
+
+         a = 4
+         if a == 4:
+             print('a = 4')
+         elif a == 5: 
+             print('a = 5')
+
+     - create an if-else statement to check if ``a`` is 4 or 5 and print result
+
    * - ``1*i``, ``1*j``,  ``1i``, ``1j``
      - ``1j``
      - complex numbers
 
    * - ``eps``
-     - ``np.spacing(1)``
-     - Distance between 1 and the nearest floating point number.
+     - ``np.finfo(float).eps`` or ``np.spacing(1)``
+     - Upper bound to relative error due to rounding in 64-bit floating point
+       arithmetic.
+
+   * - ``load data.mat``
+     - ``io.loadmat('data.mat')``
+     - Load MATLAB variables saved to the file ``data.mat``. (Note: if saving files from
+       MATLAB/Octave use a recent binary format, Python's ``loadmat`` will
+       create a dictionary with arrays and further information)
```

Comment:
(on mobile so can't select multiple lines)

There's something wrong with the grammar here. Are you trying to say "if saving from matlab using...", or "if saving from matlab, use ..."?

Reply:
trying to specify that if you're going to save arrays into the `data.mat` you have to use a binary format in MATLAB/Octave. 

how about:
`(Note: when saving arrays to ``data.mat`` in MATLAB/Octave, use a recent binary format. Python's ``loadmat`` will create a dictionary with the saved arrays and further information.)`